nemo_skills/inference/prompt/nemotron/instruct.yaml

```diff
@@ -0,0 +1,18 @@
+# Generic prompt for Nemotron instruct
```

Comment:
This is not generic, but math specific right? The generic one should probably just have question with no other instructions